ui/src/app/components/gke-app-configuration-panel.spec.tsx

```diff
@@ -107,20 +114,6 @@ describe(GKEAppConfigurationPanel.name, () => {
     await validateInitialLoadingSpinner();
   });
 
-  it('should show an error if the list apps API call fails', async () => {
```

Comment:
it doesn't do this anymore.

The component never calls list-apps directly but instead uses `maybeStartPollingForUserApps()` and `userAppsStore`

Reply:
Should we still keep this test to prove that START button remains disabled and there will be no change as userAppsStore will have an entry with ERROR state

Reply:
I'm not sure that is correct ... we can talk if you like